src/nystrum/entites.js

```diff
@@ -856,10 +856,13 @@ const Dragging = superclass => class extends superclass {
     if (!tile) return false;
     if (tile.entities.length > 0) {
       const entity = tile.entities[0];
-      if (!this.draggedEntity && entity.entityTypes.includes('DRAGGABLE')) {
+      console.log('hi there:', entity.draggability);
+      
+      if (!this.draggedEntity && entity.draggability == true) {
```

Comment:
standard is to use '===' instead of '=='. === ensures that you are equating value and type whereas == just compares value.